source/deployment-options/deploying-with-ansible/guide/install-ansible.rst

```diff
@@ -34,6 +34,7 @@ The following minimum requirements should be met to use Ansible on Windows endpo
 -  PowerShell 3.0 or newer.
 -  At least .NET version 4.0 should be installed on the Windows endpoint.
 -  A WinRM listener should be created and activated.
+-  The port 5986 specified in the `ansible_ssh_port` variable in the hosts file. 
```

Comment:
We could turn this requirement list item into a whole instruction on how to proceed configuring Ansible for Windows deployment. It could go into a new independent sentence like this

```
Before deploying on your Windows endpoints, you must set Ansible to use port `5986`. Edit the `hosts` file and add the following lines.

<<Sample configuration for  setting `ansible_ssh_port=5986` goes here>>
```

Reply:
Kindly accepted. Please review the changes done in https://github.com/wazuh/wazuh-documentation/pull/6640/commits/283a9496863649a0981b050c11b1e5ecedb03b2d.